test/parallel/test-worker-max-count-auto.js

```diff
@@ -0,0 +1,19 @@
+// Flags: --expose-internals --max-worker-threads=-1
+'use strict';
+
+// Check that when --max-worker-threads is negative,
+// the option value is auto-calculated based on the
+// number of CPUs
```

Comment:
Nit: (here and below)
```suggestion
// number of CPUs.
```